tests/integration/apps/test_data/test_transactions.py

```diff
@@ -0,0 +1,1570 @@
+import datetime
+from dateutil.tz import tzutc
+import pickle
+
+txn_count = 60
+
+test_transactions_lst = [
+    {
+        "counterpartyId": None,
+        "instrumentIdentifiers": {"Instrument/default/LusidInstrumentId": "CCY_USD"},
+        "settlementDate": datetime.datetime(2020, 2, 14, 0, 0, tzinfo=tzutc()),
+        "totalConsideration": {"amount": 10, "currency": "USD"},
+        "transactionCurrency": "USD",
+        "transactionDate": datetime.datetime(2020, 2, 14, 0, 0, tzinfo=tzutc()),
+        "transactionId": str(x) + "-rjhTransactionsTest",
+        "transactionPrice": {"price": 1.0, "type": "Price"},
+        "type": "FundsIn",
+        "units": 10,
+    }
+    for x in range(txn_count)
+]
+
+test_batched_data = [
+    [
+        "0-rjhTransactionsTest",
```

Comment:
Can we create this programmatically in a list comprehension? would make this more readable

Reply:
Discussed with Jack and not really as programmatically generating this data would essentially require a function very similar to that being testing which would then invalidate the test